source/_integrations/shelly.markdown

```diff
@@ -271,6 +271,8 @@ shelly:
   coap_port: 12345
 ```
 
+Additionally, make sure the firewall or network boundary allows 5683 (or the customized port) on the udp protocol.
```

Comment:
This is not a normal use case, but an advanced networking case. An average user reading will be confused or has no idea what to do with this, they won't be able to apply it on their Home Assistant OS-based system (which is the bigger majority of our users).

IMHO, we should not add this for that reason.

Reply:
I was waiting for another review because I had the same feeling it is to confusing for most users, I think if we change:
```
In some cases, it may be needed to customize the CoAP port
```
to:
```
In some cases, it may be needed to customize the CoAP UDP port
```
It may be enough not to confuse average users, but still allow advanced users to understand they need to map UDP port.

Reply:
Good point regarding the advanced level of networking. In the end, the issue was also quickly resolved by just googling the protocol of CoAP.
The suggestion made by @thecode would've probably saved me 5 minutes and a quick search.

Reply:
Agree, could you adjust the PR accordingly, @BerendWouters?

Reply:
I've changed the PR.